ClientApp/src/components/reuseables/ProjectEvaluationResults/ProjectEvaluationResults.tsx

```diff
@@ -101,11 +94,10 @@ const EvaluationDropdown: React.FC<EvaluationDropdownProps> = ({
         <Dropdown.Menu>
           {sections?.map(
             ({
-              employeeReviewee: { employeeId: id, firstName, lastName },
-            }: ProjectEvaluationEmployeeResult): JSX.Element => {
-              const name = fullName(firstName, lastName);
-              // TODO: update the project role when backend model supports it
-              const itemTitle = `${name} - Project Role`;
+              targetEmployeeSnippet: { employeeId: id, fullName } = {},
+              targetProjectRole: role,
+            }: EvaluationResultSection): JSX.Element => {
+              const itemTitle = `${fullName} - ${role}`;
```

Comment:
To display roles properly, this should either be
```suggestion
              const itemTitle = `${fullName} - ${transformProjectRole(role as ProjectRole)}`;
```
or...
```suggestion
              const itemTitle = `${fullName} - ${transformProjectRole(role)}`;
```
For the latter, the api would need to be updated to return a `ProjectRole` rather than a string.

Reply:
Thanks!